aspnet-core/src/AbpCompanyName.AbpProjectName.Core/AbpProjectNameCoreModule.cs

```diff
@@ -33,6 +34,8 @@ public override void PreInitialize()
             AppRoleConfig.Configure(Configuration.Modules.Zero().RoleManagement);
 
             Configuration.Settings.Providers.Add<AppSettingProvider>();
+            
+            Configuration.Localization.Languages.Add(new LanguageInfo("fa", "فارسی", "famfamfam-flags ir"));
```

Comment:
Hi.
Thanks for the corrections you have made. 
Could you please explain why did you add this line? 
Why adding the language to DefaultLanguagesCreator was not enough? 

Thanks

Reply:
Hi @faridprogrammer,
You can follow the steps to add a new language to your project from here https://aspnetboilerplate.com/Pages/Documents/Localization.

The reason you won't see the same implementation for other languages is probably because those implementations have already been made elsewhere in abp.

Reply:
Thanks for the explanation.